FFCDemoPaymentService.Tests/MessagingTests/UnitTest1.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.ComponentModel;
+using System.IO;
+using NUnit.Framework;
+using Moq;
+using FFCDemoPaymentService;
+using FFCDemoPaymentService.Messaging;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace FFCDemoPaymentService.UnitTests.MessagingTests
+{
+
+    public class AmqpConnectionTests
+    {
+        Mock<MessageService> messageService = new Mock<MessageService>();
+        Mock<AmqpConnection> aConnection = new Mock<AmqpConnection>();
+        MessageConfig messageConfig = new MessageConfig();
+
+
+        [SetUp]
+        public void Setup()
+        {
+            messageConfig.Host = "123";
+            messageConfig.Port = 111;
+            messageConfig.UseSsl = true;
+            messageConfig.PaymentQueue = "Test_Queue";
+            messageConfig.PaymentUserName = "Test_User";
+            messageConfig.PaymentPassword = "Test_Password";
+        }
+
+        [Test]
+        public void Test1()
```

Comment:
Test names should be meaningful and describe what they are trying to test.  Though these seem to be AMQP tests outside the scope of this PR so not sure why they are included.

Reply:
removed file